doc/uftrace-script.md

```diff
@@ -94,7 +95,7 @@ COMMON ANALYSIS OPTIONS
 SCRIPT EXECUTION
 ================
 The uftrace tool supports script execution for each function entry and exit.
-The supported script is only Python 2.7 as of now.
+The supported script types are Python 2.7 and LuaJIT as of now.
```

Comment:
To be precise, it should be Lua 5.1 right?

Reply:
Yes. should I change to lua5.1?

Reply:
Please.

Reply:
Updated to Lua 5.1